src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -89,7 +89,7 @@ public void onBuildTriggered(AbstractProject<?, ?> project, String commitSha, bo
         String triggeredStatus = getDescriptor().getTriggeredStatusDefault(this);
 
         // check if we even need to update
-        if (triggeredStatus.equals("--none--")) {
+        if (triggeredStatus != null && triggeredStatus.equals("--none--")) {
```

Comment:
There is a default, so it should check if the variable is not null and is equal to ```--none--```